variables/shared.yaml

```diff
@@ -58,3 +58,20 @@ variables:
 
   - name: gh_app_private_key_file_name
     value: hmcts-ado-cnp-app.private-key.pem
+
+
+  - name: vhSQLServerName
+    value: vh-infra-core-${{ parameters.env }}
+  - name: BookingsApiDbName
+    value: vhbookings
+  - name: vhResourceGroup
+    value: vh-infra-core-${{ parameters.env }}
+
+  - name: efContextName
+    value: BookingsDbContext
+  - name: efProjectName
+    value: BookingsApi.csproj
+  - name: vhKeyVault
+    value: vh-infra-core-${{ parameters.env }}
+  - name: vhSqlUsername
+    value: hvhearingsapiadmin
```

Comment:
Can this dynamically come from the key vault incase infra changes this

Reply:
username is now being retrieved from KV